passage/src/androidTest/java/id/passage/android/ChangeContactTests.kt

```diff
@@ -0,0 +1,108 @@
+package id.passage.android
+
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import id.passage.android.IntegrationTestConfig.Companion.AUTH_TOEKN
+import id.passage.android.IntegrationTestConfig.Companion.CURRENT_USER
+import id.passage.android.exceptions.PassageUserRequestException
+import id.passage.android.exceptions.PassageUserUnauthorizedException
+import junit.framework.TestCase.assertNotNull
+import junit.framework.TestCase.assertTrue
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.assertNotEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class ChangeContactTests {
+    private lateinit var passage: Passage
+
+    @Before
+    fun setUp() {
+        activityRule?.scenario?.onActivity { activity ->
+            activity?.let {
+                passage = Passage(it, IntegrationTestConfig.APP_ID_OTP)
+                passage.overrideBasePath(IntegrationTestConfig.API_BASE_URL)
+            }
+        }
+    }
+
+    @get:Rule
+    var activityRule: ActivityScenarioRule<TestActivity?>? =
+        ActivityScenarioRule(
+            TestActivity::class.java,
+        )
+
+    @Test
+    fun testChangeEmail() =
+        runBlocking {
+            // make sure we have an authToken.
+            assertNotEquals(AUTH_TOEKN, "")
+            try {
+                Passage.setAuthToken(AUTH_TOEKN)
+                val date = System.currentTimeMillis()
+                val identifier = "authentigator+$date@passage.id"
+                val response = CURRENT_USER.changeEmail(identifier)
+                assertNotNull(response.id)
+            } catch (e: Exception) {
+                assertTrue(false)
```

Comment:
For failing a test, lets use the `fail` method, as its a bit easier to read.
```suggestion
                fail("Test failed due to unexpected exception: ${e.message}")
```

Reply:
This applies in other tests in the file as well.